src/model.jl

```diff
@@ -34,35 +34,74 @@ function _make_branch_map(case::Case)::SparseMatrixCSC
     branches_to = sparse(branch_to_idx, branch_idx, 1, num_bus, num_branch)
     branches_from = sparse(branch_from_idx, branch_idx, -1, num_bus, num_branch)
     branch_map = branches_to + branches_from
+    return branch_map
 end
 
 
 """
-    _make_bus_demand(case)
+    _make_bus_demand_weighting(case)
 
-Given a Case object, build a matrix of demand by (bus, hour) for this interval.
+Given a Case object, build a sparse matrix that indicates the weighting of each bus in 
+    each zone.
 """
-function _make_bus_demand(case::Case, start_index::Int, end_index::Int)::Matrix
+function _make_bus_demand_weighting(
+    case::Case, start_index::Int, end_index::Int
+)::SparseMatrixCSC
```

Comment:
This function does not appear to actually need the `start_index` or `end_index` parameters. These are only used in `_make_bus_demand` to trim the entire profile duration down to the simulation window.

Reply:
Done.